qaevents.md

```diff
@@ -0,0 +1,116 @@
+## Main Endpoint
+**GET /api/integration/qaevents**
+
+_Unsupported._ The suggestions can be retrieved only by source and target or via direct link, see the single entry and search method below. 
+
+## GET Single suggestion
+** GET /api/integration/qaevents/<:qaevent-id>
+
+Return a single suggestion:
+
+```json
+{
+  "id":"123e4567-e89b-12d3-a456-426614174000",
+  "originalId": "oai:www.openstarts.units.it:10077/21486",
+  "title":"Index nominum et rerum",
+  "trust":"0.375",
+  "eventDate": "2020/10/09 10:11 UTC",
+  "status": "PENDING",
+  "message" : {
+    "type":"doi",
+    "value":"10.18848/1447-9494/cgp/v15i09/45934",
+    "abstracts":"Abstract Complex functionality of the ...",
+    "acronym":"EPSILON",
+    "code":"277749",
+    "funder":"EC",
+    "fundingProgram":"FP7",
+    "jurisdiction":"EU",
+    "title":"Elliptic Pdes and Symmetry of Interfaces and Layers for Odd Nonlinearities"
+  },
+  type: "qaevent"
+}
+```
+
+Attributes
+* the *id* attribute is the event primary key
+* the *originalId* attribute is the oai identifier of the target publication
+* the *title* attribute is the title of the publication as provided by the correction's source
+* the *trust* attribute is the level of accuracy of the suggestion
+* the *status* attribute is one of (ACCEPTED, REJECTED, DISCARDED, PENDING)
+* the *eventDate* attribute is the timestamp of the event reception
+* the *message* attribute is a json object which structure depends on the source and on the topic of the event. When the "topic" type is
+    * ENRICH/MISSING/PID and ENRICH/MORE/PID: fills `message.type` with the type of persistent identifier (doi, pmid, etc.) and `message.value` with the corresponding value
+    * ENRICH/MISSING/ABSTRACT: fills `message.abstract`
+    * ENRICH/MISSING/SUBJECT/ACM: fills the `message.value` with the actual keywords, the subject classification is defined by the last part of the topic (ACM, JEL, DDC, etc.)
+    * ENRICH/MISSING/PROJECT: fills `acronym`, `code`, `funder`, `fundingProgram`, `jurisdiction` and `title`
+
+Exposed links:
+* topic: link to the topic to which the event belong to
+* target: link to the item that represent the targe to whom the suggestion apply
+* related: link to a second item that is involved in the qa events (i.e. the project item)
```

Comment:
This is the first time a "project item" is mentioned in these docs.  Could we better describe what this second item is?  Do all "qaevents" only act on Publications which have related Projects?  Or is this `related` link optional?

Or should this link down to the POST section below about "associating a related item"?  If so, maybe we can provide more details on that endpoint about when a related item is used.